man/tpm2_policysecret.1.md

```diff
@@ -0,0 +1,69 @@
+% tpm2_policysecret(1) tpm2-tools | General Commands Manual
+%
+% OCTOBER 2018
+
+# NAME
+
+**tpm2_policysecret**(1) - Enables secret(password/ hmac) based authorization to a policy.
+
+# SYNOPSIS
+
+**tpm2_policysecret** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policysecret**(1) Enables secret (password/ hmac) based authorization to a policy. The secret is the authentication value of any TPM object (NV/ Hierarchy/ Loaded/ Persistent Handles).
+
+# OPTIONS
+
+  * **-c**, **--context**=_OBJECT_CONTEXT_:
+
+    A context object specifier of a transient/ permanent/ persistent object. Either a file path of a object context blob or a loaded/ persistent/ permanent handle id. See section "Context Object Format".
+
+  * **-p**, **--auth-value**=_AUTH\_VALUE_:
+
+    The authentication value of the object whose secret will be referred in the policy. Authentication values should follow the "authorization formatting standards", see section "Authorization Formatting".
```

Comment:
Is that true? Won't the only secret be a string? What about binary data?

Doesn't the auth values include things like:
"foo"
"hex:aa11bb"
"session:session.ctx"

session would be unsupported.

Reply:
Make this an argument, and if it's missing read from stdin.